app/services/GraylogService.scala

```diff
@@ -49,7 +49,18 @@ class GraylogService @Inject() extends LogService {
           val messagesjsonValue = responseJsonValue.get("messages")
           messagesjsonValue.foreach(elem => {
             val elemMap = elem.asInstanceOf[java.util.HashMap[String, java.util.HashMap[String, String]]]
-            val message: String = elemMap.get("message").get("message")
+            var message: String = elemMap.get("message").get("message")
```

Comment:
what if message does not exist in the map
can we just use regex to search/replace

Reply:
if the messages is not empty. message is not empty. If we worry about the exception, we do try catch around it.